templates/microcloud/resources.html

```diff
@@ -53,10 +52,65 @@ <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">
               <a href="https://ubuntu.com/engage/introduction-to-micro-clouds">Introduction to MicroClouds</a>
             </h3>
             <p>
-              A micro cloud is a new class of infrastructure for on-demand computing at the edge. In this webinar we go over what MicroClouds are and why they should be a part of your edge strategy, finishing with a demo.
+              MicroCloud is the newest addition to Canonical’s cloud infrastructure portfolio. It is a low-touch, open-source cloud solution purpose-built for scalable clusters and edge deployments for all types of enterprises. In this webinar, we go over what MicroClouds are and why they should be a part of your edge strategy, finishing with a demo.
+            </p>
+          </div>
+
+          <div class="col-3 col-medium-2 p-section--shallow">
```

Comment:
It would be better to add p-section--shallow to all the cards together, like put them in one div and add the class